src/libcrun/linux.c

```diff
@@ -609,6 +609,7 @@ do_mount_cgroup_v1 (libcrun_container_t *container,
   cleanup_close int tmpfsdirfd = -1;
   char *saveptr = NULL;
   bool has_cgroupns = false;
+  int rootfsfd = get_private_data (container)->rootfsfd;
```

Comment:
Seems like this variable is not used anywhere?

Reply:
yes, thanks.  I've pushed a fixed version